app/src/main/java/com/kickstarter/libs/utils/AnimationUtils.kt

```diff
@@ -20,4 +26,52 @@ object AnimationUtils {
     animation.fillAfter = true
     return animation
   }
+
+  @JvmOverloads
+  fun fadeInAndScale(view: View, duration: Long = 500L, startDelay: Long = 500L,
+                     interpolator: Interpolator = AccelerateDecelerateInterpolator()): ObjectAnimator {
+    val scaleX = PropertyValuesHolder.ofFloat("scaleX", 0f, 1f)
+    val scaleY = PropertyValuesHolder.ofFloat("scaleY", 0f, 1f)
+    val alpha = PropertyValuesHolder.ofFloat("alpha", 0f, 1f)
+    val animator = ObjectAnimator.ofPropertyValuesHolder(view, scaleX, scaleY, alpha)
+    animator.duration = duration
+    animator.startDelay = startDelay
+    animator.interpolator = interpolator
+    return animator
+  }
+
+  @JvmOverloads
+  fun fadeOutAndScale(view: View, duration: Long = 500L, startDelay: Long = 500L,
+                      interpolator: Interpolator = AccelerateDecelerateInterpolator()): ObjectAnimator {
+    val scaleX = PropertyValuesHolder.ofFloat("scaleX", 1f, 0f)
+    val scaleY = PropertyValuesHolder.ofFloat("scaleY", 1f, 0f)
+    val alpha = PropertyValuesHolder.ofFloat("alpha", 1f, 0f)
+    val animator = ObjectAnimator.ofPropertyValuesHolder(view, scaleX, scaleY, alpha)
+    animator.duration = duration
+    animator.startDelay = startDelay
+    animator.interpolator = interpolator
+    return animator
+  }
+
+  @JvmOverloads
+  fun crossFade(visibleView: View, hiddenView: View, crossDuration: Long = 500L, startDelay: Long = 500L,
+                      interpolator: Interpolator = AccelerateDecelerateInterpolator()): AnimatorSet {
+    val crossFadeAnimatorSet = AnimatorSet()
+    val fadeOutAndScale = fadeOutAndScale(visibleView, crossDuration, startDelay, interpolator)
+    val fadeInAndScale = fadeInAndScale(hiddenView, crossDuration, startDelay, interpolator)
+    
```

Comment:
What's our new line policy?

Reply:
For me, as long as it doesn't run the length of the screen it's good with me. What are your thoughts?